libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt

```diff
@@ -368,10 +360,13 @@ class RouterWrapperTests {
 
     @Test
     fun `route refresh fails with null requestUuid`() {
-        val route: DirectionsRoute = mockk(relaxed = true)
```

Comment:
mockk failed with an error `ailed to access class com.mapbox.api.directions.v5.models.DirectionsJsonObject$Builder from class com.mapbox.api.directions.v5.models.DirectionsRoute$Builder$Subclass28 (com.mapbox.api.directions.v5.models.DirectionsJsonObject$Builder is in unnamed module of loader org.robolectric.internal.AndroidSandbox$SdkSandboxClassLoader @58601e7a; com.mapbox.api.directions.v5.models.DirectionsRoute$Builder$Subclass28 is in unnamed module of loader net.bytebuddy.dynamic.loading.MultipleParentClassLoader @6a314f75)`

Reply:
This is surprising, do we know why?

Reply:
I would imagine developers mocking the route as well.

Reply:
Yeah, that's a great point @VysotskiVadim

Reply:
what I found https://github.com/mockk/mockk/issues/454 https://github.com/mockk/mockk/issues/724 and so on. Looks like we need to update mockk